build-utils/language-c/src/main/resources/templates/c/data-io-template-c.ftlh

```diff
@@ -67,7 +67,7 @@
 #include <plc4c/data.h>
 #include <plc4c/utils/list.h>
 #include <plc4c/spi/evaluation_helper.h>
-#include <plc4c/driver_${helper.getProtocolName()}.h>
+
```

Comment:
could be accidental newline but I guess this is the C-like import grouping? Would than expect one after time.h too